<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-B0390S4" name="Appointment_demo.Appointment_demo.Appointment" tests="8" failures="1" timestamp="11 Jul 2018 08:05:26 GMT" time="286.654" errors="0">
  <testcase name="TC0_VerifyLoginPage" time="2.923" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC1_loginAppSuccess" time="46.176" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC2_loginAppFailEmptyPass" time="38.276" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC2_loginAppFailEmptyAccout" time="40.403" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC3_loginAppFailWrongAcc" time="35.307" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC4_loginAppFailWrongPass" time="38.187" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC5_loginAppFailSpecialCharacter" time="40.088" classname="Appointment_demo.Appointment_demo.Appointment"/>
  <testcase name="TC6_FailCaseToCheckAlert" time="45.294" classname="Appointment_demo.Appointment_demo.Appointment">
    <failure type="java.lang.AssertionError" message="Cannot find element with bellow info //android.widget.TextView[contains(@text,&#039;abc&#039;)] expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Cannot find element with bellow info //android.widget.TextView[contains(@text,'abc')] expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at Appointment_demo.Appointment_demo.Appointment.VerifyLogin(Appointment.java:226)
at Appointment_demo.Appointment_demo.Appointment.TC6_FailCaseToCheckAlert(Appointment.java:197)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- TC6_FailCaseToCheckAlert -->
</testsuite> <!-- Appointment_demo.Appointment_demo.Appointment -->
